client/blocks/upe/grouped-checkout-element.js

```diff
@@ -0,0 +1,121 @@
+import { PaymentElements } from 'wcstripe/blocks/upe/upe-deferred-intent-creation/payment-elements';
+import { SavedTokenHandler } from 'wcstripe/blocks/upe/saved-token-handler';
+import {
+	getPaymentMethodsConstants,
+	PAYMENT_METHOD_AFTERPAY,
+	PAYMENT_METHOD_AFTERPAY_CLEARPAY,
+	PAYMENT_METHOD_CLEARPAY,
+} from 'wcstripe/stripe-utils/constants';
+import { getBlocksConfiguration } from 'wcstripe/blocks/utils';
+import Icons from 'wcstripe/payment-method-icons';
+import { initializeCheckoutIcons } from 'wcstripe/blocks/upe/checkout-icons';
+import WCStripeAPI from 'wcstripe/api';
+import PaymentProcessor from 'wcstripe/blocks/upe/upe-deferred-intent-creation/payment-processor';
+
+// Initialize checkout icons
+const isAdmin = getBlocksConfiguration()?.isAdmin ?? false;
+const checkoutIcons = initializeCheckoutIcons( isAdmin );
+
+const upeMethods = getPaymentMethodsConstants();
+
+/**
+ * Returns the UPE payment method element for registration.
+ *
+ * @param {string} paymentMethod The payment method name.
+ * @param {WCStripeAPI} api The Stripe API object.
+ * @param {Object} upeConfig The UPE configuration.
+ * @return {Object} The UPE payment method configuration.
+ */
+export const groupedCheckoutElement = ( paymentMethod, api, upeConfig ) => {
```

Comment:
I did this to follow the standard we have for express payment methods, and so I can include the new `singleCheckoutElement` on the following PR.